sass/directives/_headers.scss

```diff
@@ -129,34 +104,36 @@
     p {
       max-width: 28em;
       margin-bottom: 0;
-      font-size: $hero-text-size;
+      font-size: strip-units($hero-text-size) * 1rem;
```

Comment:
that is a good guess as to what is causing the higher font size. Since you've made changes to the way the headers are marked up, check to see if the problem is still there (go ahead and do the [additional cleanup that I suggested](https://github.com/cbdr/employer/pull/785#discussion_r112278579) first b/c that might take care of it)

If there's still one header where the font size is off, inspect that header in the browser to see if any parent containers are setting font size. Let me know if you want to pair